.circleci/config.yml

```diff
@@ -361,23 +361,24 @@ jobs:
 
   build_cli:
     docker:
-      - image: "cimg/go:<< pipeline.parameters.cli-go-version >>"
+      - image: "docker/tilt-releaser"
```

Comment:
This image contains necessary tools for cross compilation; I do not personally want to use tilt-releaser as it is.. I am thinking for v0 -- I can just use `kurtosis-releaser` with base image being `tilt-releaser`.

One thing I noticed is that this image is 2 GB whereas previous image was just 145mb and we CircleCI doesn't cache the layers -- as a FLUP we should do it. ( didn't look into how to do it - let me know if you guys did before?)

Reply:
if we don't want to do this ; the alternative is to have starlark-lsp as it's separate binary and exists in a separate repository. Users would need to install it separately, or we do it automatically via kurtosis cli. Ideally we could install a separate binary from the plugin ( but it might add delay to this project)

Reply:
Do you have concrete data on the downsides here? Is build being slower? If yes how many times slower? Is it going to be much more expensive? If yes by how much. IDK if you were waiting for input on this, but if it's the case I'd suggest we get more concrete data points as it's difficult to take a decision on something vague like this.

Reply:
yepp; I do not have the cost data -- let me know if you know where to look for it.

Regarding the completion time - with `tilt-releaser` it took 58s to complete and with the older docker image it took 2s to complete. Overall, `build_cli` step took `30s` difference. Though I baking `goreleaser into the image` may bring the execution time. I strongly think that docker api errors may be happening because of lack of caching on our end, as circle ci was pulling many layers in the image. @gbouv